Typescript: Input onKeyDown does not exist

Seems like the typings for the Input component lacks onKeyDown.

I have this error:

Property 'onKeyDown' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Input> & { children?: ReactNode; } & InputProps

It's the same with `onKeyUp`:

> error TS2339: Property 'onKeyDown' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Input> & Readonly<{ children?: ReactNode; }> & Rea...'

--- a/components/input/Input.d.ts
+++ b/components/input/Input.d.ts
@@ -109,6 +109,14 @@ export interface InputProps extends ReactToolbox.Props {
    */
   onFocus?: Function;
   /**
+   * Callback function that is fired when a key is pressed down.
+   */
+  onKeyDown?: Function;
+  /**
+   * Callback function that is fired when a key is released.
+   */
+  onKeyUp?: Function;
+  /**
    * Callback function that is fired when a key is pressed.
    */
   onKeyPress?: Function;

